query-engine/request-handlers/src/lib.rs

```diff
@@ -21,12 +19,3 @@ pub enum PrismaResponse {
     Single(GQLResponse),
     Multi(GQLBatchResponse),
 }
-
-impl PrismaResponse {
-    pub fn errors(&self) -> Vec<&GQLError> {
-        match self {
-            PrismaResponse::Single(ref s) => s.errors().collect(),
-            PrismaResponse::Multi(ref m) => m.errors().collect(),
-        }
-    }
-}
```

Comment:
I suppose this was dead code?

Reply:
Yes, there isn't any dynamic call to a particular prisma response